plugins/modules/device_credential_intent.py

```diff
@@ -2377,14 +2396,14 @@ def delete_device_credential(self, config):
                     function="delete_global_credential_v2",
                     params={"id": _id},
                 )
-                self.log(str(response))
+                self.log("Response for API delete_global_credential_v2: " + str(response), "DEBUG")
                 validation_string = "global credential deleted successfully"
                 self.check_task_response_status(response, validation_string).check_return_status()
                 final_response.get(item).append(_id)
                 config_itr = config_itr + 1
 
-        self.log("Delete Device Credential API input - " + str(final_response))
-        self.log("Global Device Credential Deleted Successfully")
+        self.log("Delete Device Credential API input: " + str(final_response), "DEBUG")
```

Comment:
self.log("Deleting Device Credential with the following parameters: {}".format(str(final_response)), "DEBUG")